header-image/index.md

```diff
@@ -0,0 +1,18 @@
+## What images can I use for a header?
+
+Here are the requirements and recommendations for header images:
+
+- The image format can be PNG, JPG, or GIF. JPG is recommended for pixel art; otherwise, a PNG is recommended. 
```

Comment:
PNG is always recommended for the raw uploads. The mediaserver will convert it to a better format when displaying it on the site, but PNG will result in the highest quality.